aws_security_group.tf

```diff
@@ -50,7 +50,7 @@ resource "aws_security_group" "acme_web_unrestricted" {
     to_port     = 0
     from_port   = 0
     protocol    = "-1"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["<cidr>"]
   }
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure no security groups allow ingress from 0.0.0.0/0 to MULTIPLE ports and protocols**
 
**Vulnerability**
It is recommended that no security group allows unrestricted ingress access
 
**Remediation**
Reduce the scope of the inbound rules to just the necessary scope, protocol, and ports

source: accurics-